lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,40 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
+      end
+
+      ALL = [
+        "Enable verbose logging with #{highlight('set VERBOSE true')}",
+        "View useful productivity tips with the #{highlight('tip')} highlight, or view them all with #{highlight('tip -l')}",
+        "When in a module, use #{highlight('back')} to go back to the top level prompt",
```

Comment:
Funny story: I think I only ever use the `back` command for demos if I want to pretend like I'm starting a fresh console.